jmeter/plans/apply.rb

```diff
@@ -0,0 +1,92 @@
+require 'ruby-jmeter'
+
+BASEURL = ENV.fetch('JMETER_TARGET_BASEURL')
+
+def url(path)
+  BASEURL + path
+end
+
+# 975 concurrent users, ~7 minutes per session
+test do
+  cookies clear_each_iteration: true
+  thread_count = 975
+
+  threads count: thread_count, continue_forever: true, duration: 420 do
+    #-> Sign up
+    visit name: 'Account page', url: url('/candidate/account') do
```

Comment:
should we factor in quite a few people arriving from Find with course params? That's quite a bit more expensive in terms of requests and db hits?

Reply:
As discussed IRL, this is something to cover in a followup card.